spec.md

```diff
@@ -222,14 +222,14 @@ that contains both context and data).
 * Examples:
   * authorization data
 
-### data
-* Type: Arbitrary payload
-* Description: The event payload. The payload depends on the event-type,
-  schema-url and event-type-version, the payload is encoded into a media format
-  which is specified by the content-type attribute (e.g. application/json).
-* Constraints:
-  * OPTIONAL
-
+## Event Data
+An event includes context and data. This "Event Data" section is the
+sibling to the above "Context Attributes". This is where the original
+event data info goes.
```

Comment:
LOL I didn't think you could use my words verbatim, I would have word-smithed it a little better.  :-)
How about:
```
The Data portion of a Cloud Event contains the original event data. The
exact format and location in which it will appear when serialized will vary
based on the serialization scheme being used.
```
I think this can probably replace both paragraphs.